projects/ccf-rui/src/app/modules/info/info-dialog/info-dialog.service.ts

```diff
@@ -1,7 +1,9 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
 import { DocumentationContent } from '../../../core/models/documentation';
-import * as documentation from 'raw-loader!./documentation.md';
+import { HttpClient } from '@angular/common/http';
+
+const documentationURL = './../../../../assets/documentation.md';
```

Comment:
URL should be `assets/docs/readme.md`

Reply:
Yep, I just saw this comment after I pushed